device-types/Fortinet/FS-3032E.yaml

```diff
@@ -0,0 +1,87 @@
+---
+manufacturer: Fortinet
+model: FortiSwitch 3032E
+slug: fortinet-fs-3032e
+part_number: FS-3032E
+u_height: 1
+weight: 19.34
+weight_unit: lb
+is_full_depth: false
+comments: '[Datasheet](https://www.fortinet.com/content/dam/fortinet/assets/data-sheets/FortiSwitch_Data_Center_Series.pdf)'
+airflow: front-to-rear
+console-ports:
+  - name: Console
+    type: rj-45
+module-bays:
+  - name: PSU1
+    position: '1'
+  - name: PSU2
+    position: '2'
+  - name: port1
+    position: '1'
+  - name: port2
+    position: '2'
+  - name: port3
+    position: '3'
+  - name: port4
+    position: '4'
+  - name: port5
+    position: '5'
+  - name: port6
+    position: '6'
+  - name: port7
+    position: '7'
+  - name: port8
+    position: '8'
+  - name: port9
+    position: '9'
+  - name: port10
+    position: '10'
+  - name: port11
+    position: '11'
+  - name: port12
+    position: '12'
+  - name: port13
+    position: '13'
+  - name: port14
+    position: '14'
+  - name: port15
+    position: '15'
+  - name: port16
+    position: '16'
+  - name: port17
+    position: '17'
+  - name: port18
+    position: '18'
+  - name: port19
+    position: '19'
+  - name: port20
+    position: '20'
+  - name: port21
+    position: '21'
+  - name: port22
+    position: '22'
+  - name: port23
+    position: '23'
+  - name: port24
+    position: '24'
+  - name: port25
+    position: '25'
+  - name: port26
+    position: '26'
+  - name: port27
+    position: '27'
+  - name: port28
+    position: '28'
+  - name: port29
+    position: '29'
+  - name: port30
+    position: '30'
+  - name: port31
+    position: '31'
+  - name: port32
+    position: '32'
```

Comment:
Are these network interfaces?? If so, they should be under `interfaces` not `module-bays`

Reply:
these ports are QSFP ports,  that can be filled with 40G or 100G transceivers (provided in the PR), or with splitters (4x10G or 4x25G), to be added after in another PR